src/model.py

```diff
@@ -151,6 +153,14 @@ def init_engine(self):
             AsyncEngineArgs(**self.vllm_engine_config)
         )
 
+        # Create vLLM custom Metrics
+        labels = {
+            "model": self.args["model_name"],
+            "version": self.args["model_version"],
+        }
+        logger = VllmStatLogger(labels=labels)
+        self.llm_engine.add_logger("triton", logger)
```

Comment:
Cross posting here: Question for this and `histogram` PR as well. Would this cause trouble if tritonserver was built without metrics ?

Reply:
+1: https://github.com/triton-inference-server/vllm_backend/pull/53#discussion_r1697679955